admin/audit.md

```diff
@@ -24,3 +23,33 @@ You can filter the logs displayed using the search filters available at the top:
 - **Resource Target**: The friendly name for the resource (e.g., the user with
   the email address **dev@coder.com**)
 - **User**: The user who performs the action
+
+## Actions
+
+The audit logs capture information about the following actions (those who
+[export Coder logs](../guides/admin/logging.md) will see this information under
+`message.fields.audit_log.action`):
+
+When reviewing Coder's audit logs, specifically, you will see the following
+actions included:
+
+- `auto_off`: Coder automatically turned on a workspace was at the preset time
+- `auto_start`: Coder automatically turned off a workspace due to inactivity
+- `connect`: a user connected to an existing workspace
+- `cordon`: a workspace provider became unavailable for new workspace creation
+  requests.
+- `create`: the user created a Coder entity (e.g., dev URL, image/image tag,
+  workspace, etc.)
+- `delete`: a user deleted a Coder entity (e.g., workspace or image)
+- `enqueue`: a user added a new job to the queue (e.g., workspace build, user
+  deletion, workspace deletion)
+- `login`: a user logs in via basic authentication or OIDC, with Coder
+  exchanging a token as a result
+- `open`: a user opened a workspace using an IDE through the browser
+- `ssh`: a user opened a web terminal or used SSH via `envproxy`
```

Comment:
```suggestion
- `ssh`: a user opened a web terminal or used SSH to access Coder
```

The implementation doesn't matter as a detail here, and `envproxy` was removed in our newer networking model